src/components/Coldcard/ColdcardSignatureImporter.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import PropTypes from "prop-types";
+import IndirectSignatureImporter from "../ScriptExplorer/IndirectSignatureImporter";
```

Comment:
Kind of feels like the SignatureImporters could be their own component directory.